README.md

```diff
@@ -39,14 +36,43 @@ Steps to run in locally in development mode:
 
 `go run . dot --serverConfig=rhn.conf |  dot -Tx11`
 
-### Profile
-Run with profile: `go run . -cpuprofile=cpu.prof -memprofile=mem.prof ...`
+## Build and release
+
+### 1. Update cmd version
+
+- Edit file `cmd/root.go` "Version" property to the desire version
+- On project root folder run `osc vc` to update the changes file with the release data
+- Manually update changes file with the release number for hte next release
+- commit and push to github
+
+### 2. Create tag
+
+- Create a tag with the version number using the format "v0.0.0" and push it to github
+
+### 3. Create a github release (optional)
+
+- On github create a new version release based on the previous tag
+
+### 4. OBS: project preparetion
+
+- Projects names:
+    - Uyuni: `systemsmanagement:Uyuni:Master`
+    - Head: Devel: `Galaxy:Manager:Head`
+    - Manager 4.2: `Devel:Galaxy:Manager:4.2`
+- Pakcage name: `inter-server-sync`
 
-View Profile data: `go tool pprof -web mem.prof`
+On porject working directory: 
 
-## Packaging
+1. Adapt the `_services` file to be able to download the correct tag for the version
+2. Run all services: `osc service runall`
+3. Check the the changes files is correctly updated
+4. Check spec file was correctly updated with the release version
+5. Add all files: `osc ar`
+6. Remove old version files `tar` and `osinfo` (`osc rm filename`)
+7. Commit everything with `osc commit`
 
-OBS project: https://build.opensuse.org/project/show/home:RDiasMateus:iss
+### 5. OBS: create submit requests
 
-## Service to create vendor sources
-`osc service rundisabled`
\ No newline at end of file
+Uyuni: `osc sr --no-cleanup <your_project> inter-server-sync systemsmanagement:Uyuni:Master`
+Manager Head: `iosc sr --no-cleanup openSUSE.org:<your_project> inter-server-sync Devel:Galaxy:Manager:Head`
+Manager 4.2: `iosc sr --no-cleanup openSUSE.org:<your_project> inter-server-sync Devel:Galaxy:Manager:4.2`
```

Comment:
Remember you will soon need to add 4.3 as well :-)

Reply:
Sure. Do you have the IBS repos already for manager 4.3?

Reply:
Nope. We still didn't branch 4.3.

But of course you can guess the IBS project will be `Devel:Galaxy:Manager:4.3`.

However I am also wondering if you really want to document this here, or if you should just say something like:

> Manager Head: `iosc sr --no-cleanup openSUSE.org:<your_project> inter-server-sync Devel:Galaxy:Manager:Head`
> And for each maintaned SUSE Manager version, one SR in the form: `iosc sr --no-cleanup openSUSE.org:<your_project> inter-server-sync Devel:Galaxy:Manager:X.Y`